src/AzurePipelinesToGitHubActionsConverter.Core/Conversion/StepsProcessing.cs

```diff
@@ -230,29 +230,42 @@ private AzurePipelines.Step CleanStepInputs(AzurePipelines.Step step)
 
         private GitHubActions.Step CreateDotNetCommandStep(AzurePipelines.Step step)
         {
-            if (step.inputs is null)
-                return null;//this is a misconfigured task
-            string runScript = "dotnet ";
-            if (step.inputs.ContainsKey("command") == true)
-            {
-                runScript += GetStepInput(step, "command") + " ";
-            }
-            if (step.inputs.ContainsKey("projects") == true)
+            if (step.inputs != null)
             {
-                runScript += GetStepInput(step, "projects") + " ";
+                string runScript = "dotnet ";
+                if (step.inputs.ContainsKey("command") == true)
+                {
+                    runScript += GetStepInput(step, "command") + " ";
+                }
+                if (step.inputs.ContainsKey("projects") == true)
+                {
+                    runScript += GetStepInput(step, "projects") + " ";
+                }
+                if (step.inputs.ContainsKey("packagestopack") == true)
+                {
+                    runScript += GetStepInput(step, "packagesToPack") + " ";
+                }
+                if (step.inputs.ContainsKey("arguments") == true)
+                {
+                    runScript += GetStepInput(step, "arguments") + " ";
+                }
+                //Remove the new line characters
+                runScript = runScript.Replace("\n", "");
+                GitHubActions.Step gitHubStep = new GitHubActions.Step
+                {
+                    run = runScript
+                };
+
+                return gitHubStep;
             }
-            if (step.inputs.ContainsKey("arguments") == true)
+            else
```

Comment:
@f2calv I refactored this a tiny bit, building on what you merged this morning, to return some information back to the end user. This ultimately doesn't fail the conversion and gives them information on how to resolve